src/output/file.rs

```diff
@@ -0,0 +1,289 @@
+// (c) Meta Platforms, Inc. and affiliates.
+//
+// Use of this source code is governed by an MIT-style
+// license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+use std::collections::BTreeMap;
+
+use crate::output as tv;
+use crate::spec;
+use maplit::{btreemap, convert_args};
+
+/// This structure represents a File message.
+/// ref: https://github.com/opencomputeproject/ocp-diag-core/tree/main/json_spec#file
+///
+/// # Examples
+///
+/// ## Create a File object with the `new` method
+///
+/// ```
+/// # use ocptv::output::*;
+///
+/// let file = File::new("name", "/path/to/file", false);
+/// ```
+///
+/// ## Create a File object with the `builder` method
+///
+/// ```
+/// # use ocptv::output::*;
+///
+/// let file = File::builder("name", "/path/to/file", false)
```

Comment:
example is incorrect, the spec explicitly says to use an URI

```suggestion
/// let file = File::builder("name", "file:///path/to/file", false)
```